src/bun.js/api/bun/udp_socket.zig

```diff
@@ -0,0 +1,477 @@
+const bun = @import("root").bun;
+const std = @import("std");
+const c = @cImport({
```

Comment:
this is probably the root of the macos compilation error. an alternative to this is hand writing the `extern fn` stuff.